packages/react/src/DevGrid/DevGrid.module.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --scrollbar-width: 0px;
+  --devGrid-width: 90%;
+  --devGrid-max-width: 80rem;
+  --devGrid-grid-size: 12;
```

Comment:
@rfearing we can make this responsive through a design token! let's pair up and i'll show you how.

Reply:
I also think these should match the grid columns used in Figma by site design. @danielguillan @ashygee, can we chat about this at the next sync?